### ISSUE TITLE
Sync reccuring events

### ISSUE THREAD

**Issue by Philoow** [owncloud/calendar#809]
Hello,

I would like to know if what i'have found is an issue or just my fault. 
I'am develop an Outlook plugin to sync Addressbooks and Calendars with Owncloud. All work very well except for recurring events. 

Owncloud to Outlook i get the recurring events, they are correct.
But if i send it to Owncloud (add Appointment) using sabredav OFC i can't see them in the calendar app. BUT the is an other strange thing. I can see my new recurring events if i export the calendar.

For example I exported Personal.ics and the last one was made with Outlook and sent to Owncloud

BEGIN:VEVENT
UID:
SUMMARY:test mars
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20160407T080000
DTEND;VALUE=DATE-TIME;TZID=Europe/Paris:20160407T083000
RRULE:FREQ=DAILY;INTERVAL=1;COUNT=5
END:VEVENT

But I can't see it in owncloud ? Normal ? Mistake ? Can someone tell me ?

Thank you !

**Comment by georgehrke** [comment]
because the UID is empty, which is not allowed?!

**Comment by georgehrke** [comment]
CalDAV should usually give you some answer on why it doesn't like your iCal

**Comment by Philoow** [comment]
I agree UID empty is normally not allowed but i can sync new events that rare not recurring. All works great as is said except for recurring events ...

**Comment by gaya4** [comment]
no useful comment from my side but I'd love to see more of this plugin. I have to use Outlook (emphasis on "have to") and Outlook is somewhat like a loner when it comes to working with other, non-family members.
Personally I don't have any recurring events except the "birthday" entries that are coming from the contacts, not the calendar itself.
